src/Microsoft.DocAsCode.MarkdigMarkdownRewriters/MarkdigMarkdownRenderer.cs

```diff
@@ -46,13 +46,44 @@ public override StringBuffer Render(IMarkdownRenderer render, MarkdownLinkInline
                     return $"[{text}]({href})";
 
                 default:
-                    return base.Render(render, token, context);
+                    return Render(render, token, context);
             }
         }
 
-        public override StringBuffer Render(IMarkdownRenderer render, DfmVideoBlockToken token, MarkdownBlockContext context)
+        public override StringBuffer Render(IMarkdownRenderer render, MarkdownBlockquoteBlockToken token, MarkdownBlockContext context)
         {
-            return $"[!VIDEO {token.Link}]";
+            const string BlockQuoteStartString = "> ";
+            const string BlockQuoteJoinString = "\n" + BlockQuoteStartString;
+
+            var content = StringBuffer.Empty;
+            for (var index = 0; index < token.Tokens.Length; index++)
+            {
+                var t = token.Tokens[index];
+                if (index == token.Tokens.Length - 1 && t is DfmVideoBlockToken videoToken)
+                {
+                    content += render.Render(t).ToString().TrimEnd();
+                }
+                else
+                {
+                    content += render.Render(t);
+                }
+            }
+            var contents = content.ToString().Split('\n');
+            content = StringBuffer.Empty;
+            foreach (var item in contents)
+            {
+                if (content == StringBuffer.Empty)
+                {
+                    content += BlockQuoteStartString;
+                    content += item;
+                }
+                else
+                {
+                    content += BlockQuoteJoinString;
+                    content += item;
```

Comment:
>content += item; [](start = 20, length = 16)

can move out of if? Can use String.Join?

Reply:
the content is StringBuffer instead of String?

Reply:
Can convet to StringBuffer finally. Just for simplicity.